examples/drivers/windows/otLwf/alarm.c

```diff
@@ -57,11 +63,11 @@ otPlatAlarmStop(
 void 
 otPlatAlarmStartAt(
     _In_ otInstance *otCtx, 
-    uint32_t now, 
-    uint32_t waitTime
+    otPlatAlarmTime *now,
+    otPlatAlarmTime *waitTime
     )
 {
     UNREFERENCED_PARAMETER(now);
-    LogVerbose(DRIVER_DEFAULT, "otPlatAlarmStartAt %u ms", waitTime);
-    otLwfEventProcessingIndicateNewWaitTime(otCtxToFilter(otCtx), waitTime);
+    LogVerbose(DRIVER_DEFAULT, "otPlatAlarmStartAt %u ms", waitTime->mMs);
+    otLwfEventProcessingIndicateNewWaitTime(otCtxToFilter(otCtx), waitTime->mMs);
```

Comment:
I think we will always need to round up here. If `mMs` is 0, then we will end up immediately calling back, indicating the timer fired, but it hasn't yet, so it will just schedule the same thing. Not until 1 ms has passed will it satisfy the timer.